config/openshift-trusted-cabundle.yaml

```diff
@@ -21,3 +21,4 @@ metadata:
     app.kubernetes.io/version: devel
     app.kubernetes.io/name: knative-eventing
     config.openshift.io/inject-trusted-cabundle: "true"
+    networking.knative.dev/trust-bundle: "true"
```

Comment:
is this just extra `label`, so that Knative can use the injected bits in addition ?

Reply:
Yes, this is to move away from custom patches eventually